src/main.ts

```diff
@@ -1,11 +1,42 @@
+import { GitHub, context } from "@actions/github";
 import { getInput, setFailed } from "@actions/core";
-import { context, GitHub } from "@actions/github";
+
+import { Octokit } from "@octokit/rest";
+import SizeLimit from "./SizeLimit";
+import Term from "./Term";
 // @ts-ignore
 import table from "markdown-table";
-import Term from "./Term";
-import SizeLimit from "./SizeLimit";
 
 const SIZE_LIMIT_URL = "https://github.com/ai/size-limit";
+const SIZE_LIMIT_HEADING = `## [size-limit](${SIZE_LIMIT_URL}) report`;
+
+const stateToEventMapping: { [key: string]: any } = {
+  COMMENTED: "COMMENT",
+  CHANGES_REQUESTED: "REQUEST_CHANGES"
+};
+
+async function fetchPreviousReview(
+  octokit: GitHub,
+  repo: { owner: string; repo: string },
+  pr: { number: number }
+) {
+  const reviews: Octokit.PullsListReviewsResponse = await octokit.paginate(
```

Comment:
Nice! this will return all reviews right ? since `list*` only return the first 30